index.html

```diff
@@ -2214,17 +2214,11 @@ <h2>The NDEFReader object</h2>
 
   <section><h3>Handling visibility change</h3>
     <p>
-      When the user agent <a href="https://html.spec.whatwg.org/#update-the-visibility-state">updates the visibility state</a>
-      of the <a>current settings object</a>'s <a>relevant global object</a>'s
-      <a>associated <code>Document</code></a> it must run these steps:
+      The <a href="https://html.spec.whatwg.org/#page-visibility-change-steps">page visibility change steps</a> for this specification, given |visibilityState| and |document|, are:
```

Comment:
This looks like a ReSpec spec, so I'm not sure it will work, but try `[=page visibility change steps=]` instead of a manual `<a>`?

Reply:
Yeah or use https://respec.org/xref/ to find the right way to reference it

Reply:
Very helpful, thanks!